reference/api/keys.md

```diff
@@ -4,22 +4,143 @@ The `/keys` route allows you to create, manage, and delete API keys. To use thes
 
 [Learn more about managing keys and their rights](/learn/security/master_api_keys.md).
 
+## Key object
+
+```json
+{
+  "name": "Default Search API Key",
+  "description": "Use it to search from the frontend code",
+  "key": "0a6e572506c52ab0bd6195921575d23092b7f0c284ab4ac86d12346c33057f99",
+  "uid": "74c9c733-3368-4738-bbe5-1d18a5fecb37",
+  "actions": [
+    "search"
+  ],
+  "indexes": [
+    "*"
+  ],
+  "expiresAt": null,
+  "createdAt": "2021-08-11T10:00:00Z",
+  "updatedAt": "2021-08-11T10:00:00Z"
+}
+```
+
+### `name`
+
+**Type**: String
+
+**Description**: A human-readable name for the key
+
+**Default value**: `null`
+
+### `description`
+
+**Type**: String
+
+**Description**: A description for the key
+
+**Default value**: `null`
+
+### `uid`
+
+**Type**: String
+
+**Description**: A [uuid v4](https://www.sohamkamani.com/uuid-versions-explained) to identify the API key. If not specified, it is automatically generated by Meilisearch
+
+**Default value**: N/A
+
+### `key`
+
+**Type**: String
+
+**Description**: An alphanumeric key value generated by Meilisearch by hashing the `uid` and the master key on API key creation. Used for authorization when [making calls to a protected Meilisearch instance](/learn/security/master_api_keys.md#communicating-with-a-protected-instance).
+
+This value is also used as the `{key}` path variable to [update](#update-a-key), [delete](#delete-a-key), or [get](#get-one-key) a specific key.
+
+**Default value**: N/A
+
+::: note
+Since `key` is a hash of the `uid` and master key, `key` values are deterministic between instances sharing the same configuration. This means if the master key changes, all `key` values are automatically changed.
+
+Since the `key` field depends on the master key, it is computed at runtime and therefore not propagated to dumps and snapshots. As a result, even if a malicious user comes into possession of your dumps or snapshots, they will not have access to your instance's API keys.
+:::
+
+### `actions`
+
+**Type**: Array
+
+**Description**: An array of API actions permitted for the key. `["*"]` for all actions.
```

Comment:
So, I'm not sure of the best way to put this, but I see we have a repeated warning a few times in the below table:

> "**Non-authorized `indexes` will be omitted from the response**"

I'd like to express this in some form in the description so we could remove it from the table and reduce the overall size/clutter a bit.

```suggestion
**Description**: An array of API actions permitted for the key, represented as strings. API actions are only possible on authorized [`indexes`](#indexes). `["*"]` for all actions.
```